substrate/frame/grandpa/src/mock.rs

```diff
@@ -77,7 +74,7 @@ where
 	RuntimeCall: From<C>,
 {
 	type OverarchingCall = RuntimeCall;
-	type Extrinsic = TestXt<RuntimeCall, ()>;
+	type Extrinsic = UncheckedExtrinsic<u64, RuntimeCall, (), ()>;
```

Comment:
similarly, I hope we can reduce the diff a bit in these cases.

Reply:
Done.